src/Uno.UI.Runtime.Skia.MacOS/UnoNativeMac/UnoNativeMac/UNOCursor.m

```diff
@@ -92,32 +62,78 @@ bool uno_cursor_set(CoreCursorType cursorType)
         case CoreCursorTypeUpArrow:
             // unsupported by macOS
             // TODO: provide custom cursor ?
-            as_requested = false;
+            cursor = [NSCursor resizeUpCursor]; // closest approximation
+            result = false;
             break;
         case CoreCursorTypeWait:
-            cursor = load_system_cursor(@"busybutclickable");
+            cursor = [UNOCursor waitCursor];
             break;
         case CoreCursorTypePin:
             // unsupported by macOS
             // TODO: provide custom cursor ?
```

Comment:
Should we remove these TODOs?

Reply:
Eventually, there's quite a few others in the new host.

What's not solved/completed before the initial release will be turned into separate public issues.

Right now it lives as part of a list inside a private issue.